CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+### [v2.6 _(Jan 15, 2019)_](https://github.com/omise/omise-magento/releases/tag/v2.6)
+
+### ✨ Highlights
+- Model\Ui\CapabilitiesConfigProvider: returns an empty array instaed of null when Installment payment method is disabled. (PR [#174](https://github.com/omise/omise-magento/pull/174))
```

Comment:
Guess this one should be in `Bug Fixes` category more than Highlights..

Reply:
well ok. will change that. I thought it is only highlight for this release. but will change it to bugfix.